test/non-functional/auth/my-privileges/global-roles/global-admin/check.it-spec.ts

```diff
@@ -0,0 +1,132 @@
+import { removeHub } from '@test/functional-api/integration/hub/hub.request.params';
+import {
+  deleteOrganization,
+  getOrganizationData,
+} from '@test/functional-api/integration/organization/organization.request.params';
+import { entitiesId } from '@test/functional-api/zcommunications/communications-helper';
+import { createOrgAndHub } from '@test/functional-api/zcommunications/create-entities-with-users-helper';
+import { TestUser } from '@test/utils';
+import { uniqueId } from '@test/utils/mutations/create-mutation';
+import { cgrud_authRes_sortedPrivileges, sortPrivileges } from '../../common';
+
+const organizationName = 'auth-ga-org-name' + uniqueId;
+const hostNameId = 'auth-ga-org-nameid' + uniqueId;
+const hubName = 'auth-ga-eco-name' + uniqueId;
+const hubNameId = 'auth-ga-eco-nameid' + uniqueId;
+
+beforeAll(async () => {
+  await createOrgAndHub(organizationName, hostNameId, hubName, hubNameId);
+});
+afterAll(async () => {
+  await removeHub(entitiesId.hubId);
+  await deleteOrganization(entitiesId.organizationId);
+});
+
+describe.skip('ChallengeAdmin - Assign / Remove Mutation', () => {
+  test.each`
+    operations                      | expected
+    ${'authorization.myPrivileges'} | ${cgrud_authRes_sortedPrivileges}
+  `('$operation', async ({ operations, expected }) => {
+    const a = '{"authorization":{"myPrivileges":""}}';
+
+    const b = JSON.parse(a);
+    const response = await getOrganizationData(
+      entitiesId.organizationId,
+      TestUser.GLOBAL_ADMIN
+    );
+    console.log(response.body);
+    // console.log(response.body.data.organization);
+
+    //   const a = String(operations);
+    const data = response.body.data.organization.b;
+    console.log(data);
+    // const jsonEvents = JSON.stringify(data);
+    //const ole = jsonEvents.a;
+    // console.log(jsonEvents);
+
+    // Assert
+    expect(data.sort()).toEqual(expected);
+  });
+});
+
+describe.skip('myPrivileges', () => {
```

Comment:
Is this skipped on purpose?

Reply:
My fault, should have not pushed this file - I wanted to test some approaches with it.